lntest/harness.go

```diff
@@ -389,7 +389,7 @@ func (n *NetworkHarness) RegisterNode(node *HarnessNode) {
 func (n *NetworkHarness) connect(ctx context.Context,
 	req *lnrpc.ConnectPeerRequest, a *HarnessNode) error {
 
-	syncTimeout := time.After(15 * time.Second)
+	syncTimeout := time.After(60 * time.Second)
```

Comment:
Any insights as to why this had to be increased?

Reply:
Because `bitcoind` was slower to catch up block hashes IIRC. Since this should be faster now I can try to revert this.